src/cryptoadvance/specter/util/fee_estimation.py

```diff
@@ -69,5 +160,20 @@ def get_fees(specter, config):
         "minimumFee": int(
             (float(specter.estimatesmartfee(20).get("feerate", 0.00001)) / 1000) * 1e8
         ),
-        "failed": "feerate" not in specter.estimatesmartfee(1),
     }
+    if "feerate" not in specter.estimatesmartfee(1):
+        # regtest does not seem to have a reasonable result for estimatesmartfee.
+        # The error-message is covering "transactions" and so the cypress-tests are not succeeding
+        # Not a perfect fix but better than to fix it everywhere in the test-code:
+        if specter.chain != "regtest":
+            fee_estimation_result.add_error_message(
+                "There was an issue while fetching fee estimation with  Bitcoin core. Please use manual fee estimation"
+            )
+            try:
+                for error in specter.estimatesmartfee(1)["errors"]:
+                    fee_estimation_result.add_error_message(error)
+            except Exception as e:
+                logger.exception(e)
```

Comment:
Can we simplify it and avoid extra rpc call as well?

```suggestion
    fee_estimate = specter.estimatesmartfee(1)
    if "feerate" not in fee_estimate:
        # regtest does not seem to have a reasonable result for estimatesmartfee.
        # The error-message is covering "transactions" and so the cypress-tests are not succeeding
        # Not a perfect fix but better than to fix it everywhere in the test-code:
        if specter.chain != "regtest":
            fee_estimation_result.add_error_message(
                "There was an issue while fetching fee estimation with  Bitcoin core. Please use manual fee estimation"
            )
            for error in fee_estimate.get("errors", []):
                fee_estimation_result.add_error_message(error)
```